examples/all-signals/README.md

```diff
@@ -0,0 +1,79 @@
+# Getting Started with OpenTelemetry Tracing in Rust
```

Comment:
On SIG meeting today, there is some support in favor of this.
Eventually, this tutorial will build on to add other signals and show advanced topics like exemplars/cross-signal correlation etc.